services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -229,6 +236,11 @@ export const ReleaseDialog: React.FC<ReleaseDialogProps> = (props) => {
                             <div className={classNames('release-dialog-author', className)}>
                                 {release?.sourceAuthor ? 'Author: ' + release?.sourceAuthor : ''}
                             </div>
+                            <div
+                                className={classNames(
+                                    'release-dialog-app',
+                                    className
+                                )}>{`Service: ${app} | Team: ${team}`}</div>
```

Comment:
Can you remove the "| Team: " if there is no team?

Reply:
I think to keep it more consistent `No team` is actually necessary. (this option is there in the team selection filter anyway).